lib/rust/ensogl/app/theme/hardcoded/src/lib.rs

```diff
@@ -368,6 +368,17 @@ define_themes! { [light:0, dark:1]
         default_x_gap_between_nodes     = 48.0  , 48.0;
         default_y_gap_between_nodes     = 32.0  , 32.0;
         minimal_x_spacing_for_new_nodes = 150.0 , 150.0;
+        // Area around every existing node where attempts to place a new node may trigger a node
+        // alignment mechanism.
+        //
+        // The specific conditions when the alignment mechanism is triggered, as well as the
+        // algorithm used to perform the alignment, are governed by the Graph Editor.
+        alignment_area_around_node {
+            distance_above_node           = 7.0  , 7.0;
+            distance_below_node           = 50.0 , 50.0;
+            distance_to_the_left_of_node  = 5.0  , 5.0;
+            distance_to_the_right_of_node = 5.0  , 5.0;
+        }
```

Comment:
Nice description here! Regarding the names – you have parent name `alignment_area_around_node`, so the children name can drop "distance" part, because `alignment_area_around_node::distance_above_node` means always exactly the same as `alignment_area_around_node::above_node`.